crates/meilisearch/tests/documents/add_documents.rs

```diff
@@ -1568,7 +1568,7 @@ async fn error_document_field_limit_reached_over_multiple_documents() {
     let (response, code) = index.update_documents(documents, Some("id")).await;
```

Comment:
Same above and in the previous tests as well.
We can share the instance with a unique index